src/net/relay.rs

```diff
@@ -804,18 +796,14 @@ impl Relayer {
 
         if new_blocks.len() > 0 {
             info!("Processing newly received blocks: {}", new_blocks.len());
+            CoordinatorCommunication::announce_new_stacks_block();
         }
-        
-        // process as many epochs as we can.
-        let max_epochs = if new_blocks.len() < 1024 { 1024 } else { new_blocks.len() };
-        let receipts: Vec<_> = chainstate.process_blocks(sortdb, max_epochs)?.into_iter()
-            .filter_map(|block_result| block_result.0).collect();
 
-        if receipts.len() > 0 || network_result.uploaded_microblocks.len() > 0 {
-            Relayer::setup_unconfirmed_state(chainstate, sortdb, &receipts)?;
+        if network_result.uploaded_microblocks.len() > 0 {
+            Relayer::setup_unconfirmed_state(chainstate, sortdb)?;
         }
 
-        Ok((new_blocks.into_iter().collect(), new_confirmed_microblocks.into_iter().collect(), new_microblocks, bad_neighbors, receipts))
+        Ok((new_blocks.into_iter().collect(), new_confirmed_microblocks.into_iter().collect(), new_microblocks, bad_neighbors, vec![]))
```

Comment:
Should we strip`Vec<StacksEpochReceipt>` from the returned type?

Reply:
Yes, we should

Reply:
Yes, I think so

Reply:
Addressed in 98edb3cd9